shuup/admin/static_src/base/scss/shuup/navigation.scss

```diff
@@ -121,7 +121,7 @@
     .nav-menu-button {
         padding: 0;
         margin: 0;
-        display: block;
+        display: block !important;
```

Comment:
Why?

Reply:
Thought it was needed but was obviously wrong. The only thing that was blocking the button from showing was the bootstrap d-lg-none.